src/components/Editor.jsx

```diff
@@ -43,66 +43,67 @@ const makeMarker = () => {
 };
 
 const pushValueToCodeMirror = _.once((value, codeMirrorInst) => {
-  if (codeMirrorInst.getValue() === '' && value !== '') {
+  if (codeMirrorInst.getValue() && value) {
```

Comment:
I think you want to flip the value of `codeMirrorInst.getValue()`.

E.g. `codeMirrorInst.getValue() === ''` => `!codeMirrorInst.getValue()`